Essentials/src/com/earth2me/essentials/commands/Commandkitreset.java

```diff
@@ -0,0 +1,66 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.Kit;
+import com.earth2me.essentials.User;
+import org.bukkit.Server;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandkitreset extends EssentialsCommand {
+    public Commandkitreset() {
+        super ("kitreset");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        if (user.isAuthorized("essentials.kit.reset")) {
```

Comment:
Essentials already checks the permissions according to the name passed in the constructor - so this would require users to have both `essentials.kitreset` and `essentials.kit.reset` in order to trigger this action, which seems a little strange.

Reply:
Plus if they don't have permission... nothing happens. This is bad UX.